### ISSUE TITLE
Decrypter Processed 0 Files

### ISSUE THREAD

**Issue by Juice805** [SpikedCola/DownloadCourseSmartBooks#6]
Thanks for your hard work on this project. I'm having an issue with my ebook. I've downloaded the files to my cache, and have pulled the JPG files from it (along with the HTML as a test). I then run the decrypter, but every time the program simply says its "Done! Processed 0 files." The Cache files are not named as in your readme, ex) "3CD7FDBFDAEE4702BD5D1E630491A77BCF17054236CB6C173E02E324E51381.jpg" and the HTML files are "393&__imagepageresolution=800&userid=6006667.htm". There are none that follow the described pattern. Maybe coursesmart changed their method? Am i doing something wrong?

Thanks again
-Juice

**Comment by soorr** [comment]
From what I understand you don't take the jpg files but the hex files under the url he specified. However my decrypter is throwing an error so coursemart may have changed their encryption method. Not sure.

**Comment by ian5142** [comment]
Yes, Coursesmart changed their encryption method. Someone on the other issue thread knows the solution but will not give it to us publicly and will not provide his e-mail either. They are scrambled JPEG files, you can open them fine, but all you will see is a mixed up page, parts here, parts there. Take a look near the bottom of the long issue thread.
